src/styles/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+.product {
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.product__image {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 40px;
+}
+
+.product__name {
+  margin-bottom: 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.product__code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+}
+
+.product__stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.rating__feedback {
+  display: flex;
+  align-items: center;
+  font-size: 10px;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.product__price {
+  margin-top: 6px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+}
+
+.price__name {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.price__value {
+  font-size: 16px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.product__button {
+  height: 40px;
+  margin-top: 16px;
+  border-radius: 5px;
+  background-color: #00acdc;
+
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+  color: #fff;
+}
+
+.product__button:hover {
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

Comment:
Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover. You could add transparent border in line 74 for example.


Before sending task to review please check it according to  [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) It's a 9 point in checklist.